packages/ui/src/components/page/SideNotification.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import styled from 'styled-components'
+import { BorderRad, Colors, Shadows, Animations } from '../../constants'
+import { CloseButton } from '../../components/buttons'
+import { Text } from '../../components/typography'
+
+interface NotificationProps {
+  title: string
+  icon?: React.ReactElement | React.ReactNode
+  message?: string | React.ReactElement | React.ReactNode
+  link?: string
+  onClick: () => void
```

Comment:
@ClumsyVlad Could you also add [the error type](https://www.figma.com/file/GlgN8uBRtvtMJtiOsdtDF7/Design?node-id=628%3A323672)?

For now, it can be some boolean value, I don't recall if we're going to have more than 2 types.

Reply:
isError added